pkg/node/node.go

```diff
@@ -259,12 +298,11 @@ func (s *CSINodeService) NodeStageVolume(ctx context.Context, req *csi.NodeStage
 	}
 
 	targetPath := getStagingPath(ll, req.GetStagingTargetPath())
-
-	isFakeAttachNeed := false
+	isFakeAttachAnnotation, isFakeAttachDR := s.isFakeAttachNeed(volumeCR)
+	isErr := false
 	ignoreErrorIfFakeAttach := func(err error) {
-		if s.isPVCNeedFakeAttach(volumeID) {
-			isFakeAttachNeed = true
-		} else {
+		isErr = true
+		if !isFakeAttachAnnotation {
```

Comment:
`isFakeAttachDR` overlaps `isFakeAttachAnnotation`, when `isFakeAttachDR` is `true` `isFakeAttachAnnotation` must be `true` as well